cylc/uiserver/scripts/gui.py

```diff
@@ -51,8 +54,15 @@
 
 def main(*argv):
     init_log()
+    GLOBAL_CONFIG = glbl_cfg().get(['hub', 'url'])
     jp_server_opts, new_gui, workflow_id = parse_args_opts()
     if '--help' not in sys.argv:
+        if GLOBAL_CONFIG:
+            print(f"Running on {GLOBAL_CONFIG} as specified in global config.")
```

Comment:
Spotted one small issue.

If you have `[hub]url` configured and run `cylc gui --help`, you'll get the standard CLI help which documents a bunch of options. However, these options only work when you're starting a standalone server, and do nothing when we're opening the hub URL in a web browser.

The best thing I can think to do is to replace the `--help` when `[hub]url` is configured which something along the lines of:

```
cylc gui [WORKFLOW]

Open the Cylc GUI in a new web browser tab.

If WORKFLOW is specified, the GUI will open on this workflow.

This command has been configured to use a centrally configured Jupyter Hub instance
rather than start a standalone server. To see the configuration options for the server
run "cylc gui --help-all", these options can be configured in the Jupyter configuration
files using "c.Spawner.cmd", see the Cylc and Jupyter Hub documentation for more
details.
```